src/components/FormatMessage/FormatMessage.vue

```diff
@@ -126,6 +128,12 @@ export default {
   },
 
   methods: {
+    containsOnlyEmojis(message) {
+      const div = document.createElement('div')
+      div.innerHTML = message.value
+      const text = div.textContent || div.innerText || ''
+      return text.length && containsOnlyEmojis(text)
```

Comment:
Quando eu olhei para isso, achei que estava em recursão, mas não, está apontado para `export const containsOnlyEmojis`. Acho que para não confudir, seria legal mudar de nome
```suggestion
      return text.length && hasOnlyEmojis(text)
```

Reply:
Esse mesmo padrão também está sendo usado para a utils `translate`